internal/schemautil/service_user.go

```diff
@@ -165,3 +157,148 @@ func TestAccCheckAivenServiceUserAttributes(n string) resource.TestCheckFunc {
 		return nil
 	}
 }
+
+// ServiceUserPasswordSchema returns the standard password schema fields for service users,
+// including support for write-only passwords.
+func ServiceUserPasswordSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"password": {
+			Type:             schema.TypeString,
+			Optional:         true,
+			Sensitive:        true,
+			Computed:         true,
+			DiffSuppressFunc: EmptyObjectDiffSuppressFunc,
+			ConflictsWith:    []string{"password_wo"},
+			Description:      "The password of the service user (auto-generated if not provided). Empty when using `password_wo`.",
+		},
+		"password_wo": {
+			Type:          schema.TypeString,
+			Optional:      true,
+			Sensitive:     true,
+			WriteOnly:     true,
+			RequiredWith:  []string{"password_wo_version"},
+			ConflictsWith: []string{"password"},
+			ValidateFunc:  validation.StringIsNotEmpty,
+			Description:   "The password of the service user (write-only, not stored in state). Must be used with `password_wo_version`. Cannot be empty.",
+		},
```

Comment:
nit: It turns out, "write only" requires Terraform >=1.11 https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/write-only-arguments. I would add that to the description just in case.